skype/skype-ps/skype/Set-CsTeamsUpgradePolicy.md

```diff
@@ -132,18 +135,18 @@ For more information, see about_CommonParameters (http://go.microsoft.com/fwlink
 
 ## RELATED LINKS
 
-[Migration and interoperability guidance for organizations using Teams together with Skype for Business](https://docs.microsoft.com/en-us/MicrosoftTeams/migration-interop-guidance-for-teams-with-skype)
+- [Migration and interoperability guidance for organizations using Teams together with Skype for Business](https://docs.microsoft.com/en-us/MicrosoftTeams/migration-interop-guidance-for-teams-with-skype)
```

Comment:
Need to undo the list formatting, used in other document formats, but the PowerShell processor doesn't like it under the RELATED LINKS header